dll/win32/newdev/wizard.c

```diff
@@ -589,7 +591,12 @@ CHSourceDlgProc(
             dwStyle = GetWindowLongPtr(hwndControl, GWL_STYLE);
             SetWindowLongPtr(hwndControl, GWL_STYLE, dwStyle & ~WS_SYSMENU);
 
-            PopulateCustomPathCombo(GetDlgItem(hwndDlg, IDC_COMBO_PATH));
+            hwndCombo = GetDlgItem(hwndDlg, IDC_COMBO_PATH);
+            PopulateCustomPathCombo(hwndCombo);
+
+            GetComboBoxInfo(hwndCombo, &info);
+            hwndEdit = info.hwndItem;
+            SHAutoComplete(hwndEdit, SHACF_FILESYS_DIRS);
```

Comment:
No need to use intermediate variables there.
```suggestion
            SHAutoComplete(info.hwndItem, SHACF_FILESYS_DIRS);
```

Reply:
OK.